native/jest.config.js

```diff
@@ -14,6 +14,8 @@ const transformNodeModules = [
 ]
 
 module.exports = {
+  rootDir: 'src',
+  roots: ['<rootDir>', '<rootDir>/../../api-client/src'],
```

Comment:
This means that jest runs both the native and the api clients tests, right? Does that mean in the ci the api client is executed twice (once for web and once for native)?